packages/foreman/rubygem-nokogiri/rubygem-nokogiri.spec

```
@@ -71,8 +71,8 @@ cp -a .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
 
 mkdir -p %{buildroot}%{gem_extdir_mri}
-cp -a .%{gem_extdir_mri}/gem.build_complete %{buildroot}%{gem_extdir_mri}/
-cp -a .%{gem_extdir_mri}/%{gem_name}/*.so %{buildroot}%{gem_extdir_mri}/
+cp -a .%{gem_extdir_mri}/* %{buildroot}%{gem_extdir_mri}/
```

I'd be more specific and copy it directly
```suggestion
cp -a .%{gem_extdir_mri}/%{gem_name}/*.so %{buildroot}%{gem_extdir_mri}/%{gem_name}
```

Normally I like that since it's easier to detect changes. You also don't accidentally copy other files like `.la` if they're present.